beetmoverscript/templates/fennec_nightly_en_us_multi_signed.yml

```diff
@@ -7,5 +7,5 @@ mapping:
   {% set locale_prefix = 'en-US/' if locale == 'en-US' else '' %}
     apk:
       artifact: {{ locale_prefix }}target.apk
-      s3_key: {{ locale_prefix  }}fennec-{{ version }}.{{ locale }}.android.arm.apk
+      s3_key: {{ locale_prefix  }}fennec-{{ version }}.{{ locale }}.android-arm.apk
```

Comment:
@lundjordan Not sure I'm right here but glancing at the Balrog release blob I've noticed this difference.